app/src/main/java/com/academy/ramon/marvelcomicviewer/MainActivity.java

```diff
@@ -49,11 +54,13 @@ public void getHeroesList(){
             public void onResponse(Call<HeroesResponse> call, Response<HeroesResponse> response) {
                 HeroesResponse heroesResponse;
                 heroesResponse = response.body();
-                rvHeroes.setAdapter(new HeroAdapter(heroesResponse.getItems(),rvHeroes.getContext()));
-                heroes = heroesResponse.getItems();
-                Log.d("hey there", response.body().toString());
-            }
+                heroes = heroesResponse.getData().getResults();
+                HeroAdapter adapter = new HeroAdapter(heroes, rvHeroes.getContext());
```

Comment:
@Mathreyu you can save the context parameter and get it from the parent group view from the get view adapter method